services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/ResumableFileDownload.java

```diff
@@ -193,90 +187,52 @@ public void writeToOutputStream(OutputStream outputStream) {
     }
 
     /**
-     * Returns the serialized JSON data representing this object as a UTF-8 string.
+     * Returns the serialized JSON data representing this object as a string.
      */
-    public String toUtf8String() {
+    public String serializeToString() {
         return new String(ResumableFileDownloadSerializer.toJson(this), StandardCharsets.UTF_8);
     }
 
-    /**
-     * Returns the serialized JSON data representing this object as a string encoded with the supplied charset.
-     *
-     * @param cs encoding charset
-     * @return the serialized JSON string
-     */
-    public String toString(Charset cs) {
-        return new String(ResumableFileDownloadSerializer.toJson(this), cs);
-    }
```

Comment:
Should we override toString using ToStringBuilder?

Reply:
This is already done, higher up in the file.